### ISSUE TITLE
Gemspec borked on v2.1.3-rc1

### ISSUE THREAD

**Issue by tlubz** [net-ssh/net-sftp#76]
Installing v2.1.3-rc1 using Bundler results in the following error

```
$ bundle
Fetching gem metadata from https://rubygems.org/.........
Fetching https://github.com/net-ssh/net-sftp.git

[!] There was an error while loading `net-sftp.gemspec`: cannot load such file -- net/ssh/version
Does it try to require a relative path? That's been removed in Ruby 1.9. Bundler cannot continue.

 #  from /Users/tomlubitz/.rvm/gems/ruby-2.5.1@transmogrifier/bundler/gems/net-sftp-13cba0e0d801/net-sftp.gemspec:1
 #  -------------------------------------------
 >  require_relative 'lib/net/sftp/version'
 #  
 #  -------------------------------------------
```

Digging into the code, it looks like `net/sftp/version` is requiring `net/ssh/version`, which may not be part of the load path yet. It's only guaranteed to be in the load path _after_ the gem's dependencies are installed, and the gemspec needs to be loaded in order to determine which dependencies to install.

![self destruction](https://i.imgur.com/e8a5w9s.gif)

I think the solution is to:

1. use `$LOAD_PATH.unshift File.expand_path("../lib", __FILE__)` instead of `require_relative`, though this may be a red herring. this is how it works in e.g.`rspec.gemspec`.
2. don't require `net/ssh/version` inside `net/sftp/version`. This means `Net::SFTP::Version` can't be a subclass of `Net::SSH::Version` ¯\_(ツ)_/¯

I'm not familiar enough with the history of this repo to open a PR without more information, but I may be able to make the above changes if they sound good.

tagging @mfazekas since it looks like the issue was introduced in https://github.com/net-ssh/net-sftp/commit/4ffe058c25993101443dc7d05885b7fa2f7128e7

**Comment by marco-gallardo** [comment]
Just confirming that the issue was introduced on https://github.com/net-ssh/net-sftp/commit/4ffe058c25993101443dc7d05885b7fa2f7128e7.

My application was working just fine but after doing a new deployment today the error showed up.

Error was gone when I just left `gem 'net-sftp'`, which I think it also confirms that the `require_relative` worked once dependencies were installed (previously I had `gem 'net-sftp', git: 'https://github.com/net-ssh/net-sftp.git'` on my Gemfile).

**Comment by mfazekas** [comment]
@marco-gallardo thanks for reproting the issue, yes i think the correct fix is making`Net::SFTP::Version` to stand on it's own and not to inherit - Net::SSH::Version.

**Comment by mfazekas** [comment]
@marco-gallardo can you confirm this is fixed with net-sftp-2.1.3.rc2, thanks ?
